aten/src/TH/vector/AVX2.cpp

```diff
@@ -6,7 +6,7 @@
 #include <immintrin.h>
 #endif
 #include "AVX2.h"
-#include "avx_mathfun.h"
+#include <ATen/native/cpu/avx_mathfun.h>
```

Comment:
Now `TH` depends on ATen which depends on `TH`?

Reply:
Not sure what the best thing to do is here. You guys tell me. cc @colesbury @apaszke

Reply:
It's one library now. We're not trying to keep TH separately compilable.